lib/extensions.js

```diff
@@ -260,6 +261,80 @@ function getBinaryPath() {
 }
 
 /**
+ * An array of paths suitable for use as a local disk cache of the binding.
+ *
+ * @return {[]String} an array of paths
+ * @api public
+ */
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_sass_binary_cache,
+    process.env.npm_config_cache,
+    getBinaryPath(),
```

Comment:
I'm not sure, I need to think about this. In the install script we only ever care about the final destination. The edge case is that it's possible for the cache directory and vendor directory to be one and the same.